.github/ISSUE_TEMPLATE/bug-report---.md

```diff
@@ -0,0 +1,48 @@
+---
+name: "Bug Report \U0001F41E"
+about: 'Something isn''t working as expected? Here is the right place to report. '
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+  Please fill out each section below, otherwise, your issue will be closed. This info allows maintainers to diagnose (and fix!) your issue as quickly as possible.
```

Comment:
Instead of saying `your issue would be close`, we could say something more polite like `or it would give a tough time to contributors.`